### PR TITLE
Delfin deployment on kubernetes

### PR DESCRIPTION
**Pull request by vineela1999**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>

kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:
This PR has kubernetes object files when followed the delfin-kubernetes-guide docs can be able deploy delfin on k8s

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

Able to get all the required pods up:
Screenshot from 2022-01-03 11-18-58
Able to request to a service:
Screenshot from 2022-01-03 11-18-10

**Special notes for your reviewer**:

### CODE REVIEW

File: delfin-kubernetes/delfin-alert-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        io.soda.service: delfin-alert
+    spec:
+      containers:
+        - args:
+            - alert
+          env:
+            - name: OS_COORDINATION__BACKEND_SERVER
+              value: redis:6379
+            - name: OS_DEFAULT__TRANSPORT_URL
+              value: rabbit://delfinuser:delfinpass@rabbitmq:5672//
```
Comment (skdwriting) [inline delfin-kubernetes/delfin-alert-deployment.yaml:26]: plain text?

File: delfin-kubernetes/delfin-alert-service.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  name: delfin-alert
+spec:
+  ports:
+    - name: "162"
+      port: 162
+      targetPort: 162
+  selector:
+    io.soda.service: delfin-alert
+status:
+  loadBalancer: {}
```
Comment (skdwriting) [inline delfin-kubernetes/delfin-alert-service.yaml:15]: ?

File: delfin-kubernetes/delfin-api-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        io.soda.service: delfin-api
+    spec:
+      containers:
+        - args:
+            - api
+          env:
+            - name: OS_COORDINATION__BACKEND_SERVER
+              value: redis:6379
+            - name: OS_DEFAULT__TRANSPORT_URL
+              value: rabbit://delfinuser:delfinpass@rabbitmq:5672//
```
Comment (skdwriting) [inline delfin-kubernetes/delfin-api-deployment.yaml:26]: plain text?

Comment (skdwriting) [reply delfin-kubernetes/delfin-api-deployment.yaml:26]: same for below files also.

File: delfin-kubernetes/delfin-alert-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
```
Comment (skdwriting) [inline delfin-kubernetes/delfin-alert-deployment.yaml:1]: Please add a readme in the example folder to describe what is this delfin=kubernetes. Also please provide the link to the documentation where we add the guide.

File: delfin-kubernetes/delfin-alert-service.yaml (hunk 2 above)
Comment (vineela1999) [reply delfin-kubernetes/delfin-alert-service.yaml]: changed

**Comment by sushanthakumar** [comment]
After adding readme file, you can move all yamls to "deploy" folder

File: delfin-kubernetes/delfin-alert-deployment.yaml (hunk 4 above)
Comment (vineela1999) [reply delfin-kubernetes/delfin-alert-deployment.yaml:1]: Added readme.MD

Comment (vineela1999) [reply delfin-kubernetes/delfin-alert-deployment.yaml:1]: added readme file

**Review by joseph-v — APPROVED** [review]
LGTM

File: delfin-kubernetes/delfin-alert-deployment.yaml (hunk 1 above)
Comment (vineela1999) [reply delfin-kubernetes/delfin-alert-deployment.yaml]: it will be considered in future

**Comment by sushanthakumar** [comment]
LGTM

File: delfin-kubernetes/deploy/delfin-alert-deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+      labels:
+        io.soda.service: delfin-alert
+    spec:
+      containers:
+        - args:
+            - alert
+          env:
+            - name: OS_COORDINATION__BACKEND_SERVER
+              value: redis:6379
+            - name: OS_DEFAULT__TRANSPORT_URL
```
Comment (kumarashit) [inline delfin-kubernetes/deploy/delfin-alert-deployment.yaml:25]: seems to be double _

**Review by kumarashit — APPROVED** [review]
LGTM

**Pull request merged**